### ISSUE TITLE
Parent coordinates editable?

### ISSUE THREAD

**Issue by BruttherJOE** [introlab/find-object#140]
![image](https://github.com/introlab/find-object/assets/54128914/7c2b93b9-78c1-433a-a56b-ad59f0f9ac02)

Hi, firstly, thank you for releasing this package, as it works wonderfully. I have just known about this and have begun to explore the package.

My question is, are the parent coordinates for the detected objects editable? I could not find anything about this in the .ini file or in the node bringup I did. Where is this variable referenced? hovering above it in rviz tells me that this is not editable. I would like to use the parent coordinates = camera_link or world instead if it is possible (unless someone tells me this is bad practice)

**Comment by BruttherJOE** [comment]
Ok, this is my bad and I just realised that this is a dumb question. The quick fix is here for anyone who needs it; I just realised that parent links cannot be changed. But I can define another frame and tf that with the correct orientation tf such that the detected object is rotated the right way. 

Again, I do not know if this is the best way of doing things or it is just bad practice and makes everything more confusing. In any case, my problem is solved and I will close the issue.
